CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support for `loongarch64`, `m68k`, `sheb` and `sh` architectures. Note that Rust has
   no support for `SuperH` so you can not use libseccomp-rs on such architectures.
   You can however create and export seccomp-bpf for them.
+- `SeccompError::sysrawrc()` that queries the system's raw error code directly returned
+  by [`ScmpFilterAttr::ApiSysRawRc`].
```

Comment:
Rustdoc intradoc-links are not supported inside CHANGELOG.md :wink:

Reply:
Oh, yes 🤣. Fixed.